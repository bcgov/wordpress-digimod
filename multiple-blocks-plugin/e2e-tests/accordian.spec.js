import {
    createNewPost,
    enablePageDialogAccept,
    insertBlock,
    getEditedPostContent
} from '@wordpress/e2e-test-utils';

describe( 'Accordion block', () => {
    jest.setTimeout(60000);

    beforeAll( async () => {
        await enablePageDialogAccept();
    } );
    beforeEach( async () => {
        await createNewPost();
    } );

    it( 'Accordion block should be available', async () => {
        await insertBlock( 'accordi' );
		// Check if block was inserted
        
		expect( await page.$( '[data-type="multiple-blocks-plugin/accordian"]' ) ).not.toBeNull();

		expect( await getEditedPostContent() ).toMatchSnapshot();
	} );
} );