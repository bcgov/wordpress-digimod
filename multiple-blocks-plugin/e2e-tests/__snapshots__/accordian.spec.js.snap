// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Accordion block Accordion block should be available 1`] = `
"<!-- wp:multiple-blocks-plugin/accordian -->
<div react-component=\\"CollapseStyled\\"><div react-component=\\"PanelStyled\\"><div react-component=\\"StyleRichText\\" assign-inner-content-to-prop=\\"htmlOrMarkdown\\"></div></div></div>
<!-- /wp:multiple-blocks-plugin/accordian -->"
`;
